app/src/main/java/com/willowtree/vocable/MainActivity.kt

```diff
@@ -27,7 +27,12 @@ class MainActivity : BaseActivity() {
 
         binding = ActivityMainBinding.inflate(layoutInflater)
         setContentView(binding.root)
-        binding.pointerView.isVisible = BuildConfig.USE_HEAD_TRACKING && sharedPrefs.getHeadTrackingEnabled()
+
+        if (BuildConfig.USE_HEAD_TRACKING) {
+            binding.pointerView.isVisible = true
```

Comment:
I think this value actually defaults to `true` in the `build.gradle` file. So if it's not explicitly set to false, users won't be able to turn head tracking off.

Reply:
ah i see, so should we do the other way around?

Reply:
Yeah probably best to check:
``` kotlin
if (!BuildConfig.USE_HEAD_TRACKING) {
  binding.pointerView.isVisible = false
} else {
  binding.pointerView.isVisible = sharedPrefs.getHeadTrackingEnabled()
}
```